src/outputs/embeddable-widget.js

```diff
@@ -6,16 +6,20 @@ import '../../vendor/cleanslate.css';
 export default class EmbeddableWidget {
   static el;
 
-  static mount() {
-    const component = <Widget />;
+  static mount(params=null) {
+    const component = <Widget params={params}/>;
```

Comment:
I would do something like this here:
```
static mount({parent, ...params} = {})
const component = <Widget {...params}/>
```

Example:
<img width="692" alt="image" src="https://user-images.githubusercontent.com/6145422/59785068-e916a780-9291-11e9-8fa0-c4902c6e4a5f.png">